ui/common/css/header/_topnav-hidden.scss

```diff
@@ -101,16 +101,17 @@
   }
 
   #topnav {
+    @include back-blur(6px);
```

Comment:
should this be for transparent theme only?

Reply:
checking out transparent theme, I saw this:

![574x1016_2024-03-18](https://github.com/lichess-org/lila/assets/140370/71c05d87-cc74-4d49-84dd-9e9267024be4)

Reply:
oh shit, that one slipped back in there. it's fixed now.

Try the new header sticky style as of this push and let me know if you see anything you like.

Reply:
This is great!